flutter-gui-tests/build.gradle

```diff
@@ -0,0 +1,103 @@
+plugins {
+  id 'org.jetbrains.intellij' version '0.4.4'
+  id 'org.jetbrains.kotlin.jvm' version '1.3.21'
+}
+
+group 'io.flutter.tests.gui'
+version '1.0-SNAPSHOT'
+
+repositories {
+  mavenLocal()
+  mavenCentral()
+  jcenter()
+}
+
+dependencies {
+  compile "org.jetbrains.kotlin:kotlin-reflect"
+  testCompile group: 'junit', name: 'junit', version: '4.12'
+}
+
+
+sourceSets {
+  main {
+    java.srcDir 'src'
+    kotlin.srcDir 'src'
+    resources.srcDir 'res'
+  }
+  test {
+    java.srcDir 'testSrc'
+    java.srcDir 'testsData'
+  }
+}
+
+intellij {
+  plugins 'com.intellij.testGuiFramework:0.9.42@nightly', 'Dart:191.5849.16'
```

Comment:
Will these need regular updating?

The Dart one looks a little behind latest:

https://plugins.jetbrains.com/plugin/6351-dart

Does that matter?

Reply:
Yeah, this is something that we might want to add to the plugin tool. I wonder if there is a way to simply say "use whatever is available"? I'll have to investigate.

Reply:
Sounds good!